tests/test_distributions.py

```diff
@@ -45,6 +56,16 @@ def test_json_to_distribution() -> None:
     pytest.raises(ValueError, lambda: distributions.json_to_distribution(unknown_json))
 
 
+def test_abbreviated_json_to_distribution() -> None:
+
+    for key in EXAMPLE_ABBREVIATED_JSONS:
+        distribution_actual = distributions.json_to_distribution(EXAMPLE_JSONS[key])
```

Comment:
Ops, I think you meant the following (line might become too long).
```suggestion
        distribution_actual = distributions.json_to_distribution(EXAMPLE_ABBREVIATED_JSONS[key])
```

Reply:
Exactly...